### DISCUSSION TITLE
How to prevent or suspend, specific execution of CronWorkflow?

### DISCUSSION THREAD

**Discussion by imjulioc** [argoproj/argo-workflows#9343] [Q&A] [answered]
Hello and good day everyone.

I am struggling with this problem.
Let's say a CronWorkflow with schedule: "0 0 */1 * *" (run daily at 00:00). A visual example:
![image](https://user-images.githubusercontent.com/63888039/184179492-7d02596e-a700-402f-b8b9-3ffc73147aaf.png)
Those executions were calculated with some cron parsing library.
I know, those are actually nothing, that can be referenced or retrieved in any form (similar to a registry in DB), merely calculations, when the CronWorkflow would be triggered.

I wonder if actions like these, could be achieved:
![image](https://user-images.githubusercontent.com/63888039/184183538-d835d56f-9c1c-4ad3-9ba0-7a5b4b217337.png)
_Cancel, suspend, delete or whatever related to a specific execution._
In this case, affecting only the execution on July 14, keep the other ones unaffected.

What would be your solution or recommendation?. It's actually impossible?.

At least I can represent a "Delete all" action :joy:.

**Comment by terrytangyuan** [comment, marked as answer]
You can set `suspend: true` on your CronWorkflow. If you want to suspend/cancel a particular execution, you just need to make sure `suspend` is true during that execution timeframe and then reset it back to false afterwards.

**Reply by imjulioc** [reply]
Something like a second CronWorkflow that suspends the first one?

**Reply by terrytangyuan** [reply]
Yeah or a regular CronJob.
